features/support/env.rb

```diff
@@ -16,6 +16,7 @@
 require 'slimmer/test'
 
 GovukTest.configure
+Capybara.javascript_driver = :selenium_chrome
```

Comment:
Was this an attempt to get that failing test to pass? Seems like changing this could have consequences.

Reply:
yes, for debugging - this is a WIP